spec/support/capture_std.rb

```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
-require 'active_support/core_ext/kernel/reporting'
+
+# This is necessary because Rails 5 removed the Kernel extension of
+# that added `#silence_stream` and moved it to their testing set of
+# libraries. Therefore, I'm including this here. Technically any
+# testing will conceivably install version 5 meaning the include is
+# necessary. However, this allows us to clearly be compliant with
+# both rails 4 and 5 which the gemspec supports.
+require 'active_support/gem_version'
+
+if ActiveSupport::VERSION::MAJOR == 5
```

Comment:
I would check for version 4 instead; Then there is a chance it will also work for Rails 6